src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -465,7 +465,7 @@
     <value>Members '{0}' and '{1}' on type '{2}' cannot both bind with parameter '{3}' in the deserialization constructor.</value>
   </data>
   <data name="ConstructorParamIncompleteBinding" xml:space="preserve">
-    <value>Each parameter in the deserialization constructor on type '{0}' must bind to an object property or field on deserialization. Each parameter name must match with a property or field on the object. The match can be case-insensitive.</value>
+    <value>Each parameter in the deserialization constructor on type '{0}' must bind to an object property or (when 'JsonSerializerOptions.IncludeFields' is enabled) a field, on deserialization. Each parameter name must match with a property or field on the object. The match can be case-insensitive.</value>
```

Comment:
consider structuring this sentence so that there are no brackets - especially seems that frequently that is a hint to a solution

Reply:
Rephrased with parentheses.

> Parenthetical remarks (however relevant) are unnecessary.

:)